addon/lib/document.js

```diff
@@ -171,6 +173,7 @@ export default Base.extend({
       if (this.parentDocument) return this.parentDocument.jexlContext;
 
       return {
+        null: null,
```

Comment:
As discussed, probably worthy of a comment since it is not so obvious why this is necessary.

Reply:
Done